edu.gemini.lch.services/src/main/java/edu/gemini/lch/services/resources/LaserNightTestController.java

```diff
@@ -19,15 +18,16 @@
 @RequestMapping(value="/test/nights")
 public class LaserNightTestController {
 
-    private static final DateTimeFormatter formatter = DateTimeFormat.forPattern("yyyyMMdd").withZone(DateTimeZone.UTC);
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyyMMdd").withZone(ZoneId.systemDefault());
 
     @Resource
     private LaserNightService laserNightService;
 
     @RequestMapping(value="", method= RequestMethod.GET, produces="application/xml")
     @ResponseBody
     public NightFull getTestNight(@RequestParam(value = "date") String dateString, @RequestParam Boolean full, @RequestParam Double laserLimit) {
-        DateTime date = formatter.parseDateTime(dateString).withTimeAtStartOfDay();
+//        ZonedDateTime date = formatter.parse(dateString).withTimeAtStartOfDay();
+        ZonedDateTime date = ZonedDateTime.parse(dateString, formatter);
```

Comment:
Do you get the time at start of the day automatically?

Reply:
No... I think this is when I reached end of day and had to leave in a hurry. Should be okay now.